src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1450,6 +1527,7 @@ else if("text-decoration-line".equalsIgnoreCase(element))
 		}
 		else if("text-decoration-skip".equalsIgnoreCase(element))
 		{
+			// FIXME: the specified values here are invalid
```

Comment:
were they valid once (⇒ are they accepted by browsers)? If yes, we shouldn’t delete them, because a mirrored page or an old page may (still) use them. Some of the freesites are 15 years old and can’t be updated anymore but are still useful references.

Reply:
None of these are accepted by Firefox or Chrome. The value "spaces" and may be "ink" may have worked for a short time in Chrome, but this entire property has been unrecognized by Firefox and Chrome now.

Reply:
:+1: